xla/service/gpu/cudnn_fused_mha_rewriter.cc

```diff
@@ -1381,14 +1528,15 @@ StatusOr<bool> FuseBwdMultiHeadedAttentionBlock(
     HloComputation* comp, HloInstruction* bmm_1_grad_1,
     HloInstruction* bmm_1_grad_2, HloInstruction* bmm_2_grad_1,
     HloInstruction* bmm_2_grad_2, HloInstruction* fwd_fmha_call,
-    HloInstruction* d_intermediate, HloInstruction* mask,
-    std::string& bwd_custom_call_name, bool fwd_bmm_2_canonicalized,
-    bool is_bmm2_grad1_canonicalized) {
+    HloInstruction* d_intermediate, HloInstruction* mask, HloInstruction* bias,
+    std::string& bwd_custom_call_name) {
   HloInstruction* rhs_bmm1_grad_gemm1;
   HloInstruction* lhs_bmm1_grad_gemm2;
   HloInstruction* lhs_bmm2_grad_gemm1;
   HloInstruction* rhs_bmm2_grad_gemm2;
   HloInstruction* d_output_grad;
+  HloInstruction* fwd_act;
```

Comment:
Nit: maybe move this down close to where it is assigned?
I think that makes the code easier to read.